providers/siw/siw.c

```diff
@@ -482,7 +482,7 @@ static void siw_async_event(struct ibv_context *ctx,
 static int siw_notify_cq(struct ibv_cq *ibcq, int solicited)
 {
 	struct siw_cq *cq = cq_base2siw(ibcq);
-	atomic_ulong *notifyp = (atomic_ulong *)&cq->ctrl->notify;
+	atomic_uint *notifyp = (atomic_uint *)&cq->ctrl->flags;
```

Comment:
Shouldn't this be atomic_uint32_t ?

Reply:
hmm, there is no such type for my gcc. unsigned int is 4 bytes on 32bit and 64bit arch's. so that would be OK as is? alternatively, we could call it a 'atomic_uint_fast32_t' -- which would be available....but it doesn't appear to be widely used to me.

Reply:
It is spelled  _Atomic(uint32_t) and used widely in rdma-core

Reply:
great, thanks. will change

Reply:
_Atomic(uint32_t) *notifyp = (_Atomic(uint32_t) *)&cq->ctrl->flags;
gives me a false positive from travis (seem to take the '*' as a multiplication):
>>>providers/siw/siw.c(485,): error : need consistent spacing around '*' (ctx:WxV)<<<

will cast within the atomic_store()

Reply:
Ignore checkpatch warnings that are stupid.